ArduCopter/mode_land.cpp

```diff
@@ -7,9 +7,14 @@ bool ModeLand::init(bool ignore_checks)
     control_position = copter.position_ok();
 
     // set horizontal speed and acceleration limits
-    pos_control->set_max_speed_accel_xy(wp_nav->get_default_speed_xy(), wp_nav->get_wp_acceleration());
-    pos_control->set_correction_speed_accel_xy(wp_nav->get_default_speed_xy(), wp_nav->get_wp_acceleration());
-
+    // pos_control->set_max_speed_accel_xy(wp_nav->get_default_speed_xy(), wp_nav->get_wp_acceleration());
+    // pos_control->set_correction_speed_accel_xy(wp_nav->get_default_speed_xy(), wp_nav->get_wp_acceleration());
+
+    // adjusted horizontal speed and acceleration limits as per UAV
+    const float adjusted_acceleration  = copter.g.adjusted_acl;
```

Comment:
I think if the parameter is zero (or less) we would want to fall back to using the wp_nav acceleration.